cmd/utils/global_options.go

```diff
@@ -66,23 +67,76 @@ func HorizonClientURLOption(configKey *string) *config.ConfigOption {
 	}
 }
 
+func ChannelAccountEncryptionPassphraseOption(configKey *string) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:      "channel-account-encryption-passphrase",
+		Usage:     "The Encryption Passphrase used to encrypt the channel accounts private key.",
+		OptType:   types.String,
+		ConfigKey: configKey,
+		Required:  true,
+	}
+}
+
+func DistributionAccountPublicKeyOption(configKey *string) *config.ConfigOption {
+	return &config.ConfigOption{
+		Name:           "distribution-account-public-key",
+		Usage:          "The Distribution Account public key.",
+		OptType:        types.String,
+		CustomSetValue: SetConfigOptionStellarPublicKey,
+		ConfigKey:      configKey,
+		Required:       true,
+	}
+}
+
 func DistributionAccountPrivateKeyOption(configKey *string) *config.ConfigOption {
 	return &config.ConfigOption{
 		Name:           "distribution-account-private-key",
-		Usage:          "The Distribution Account private key.",
+		Usage:          `The Distribution Account private key. It's required if the configured signature client is "ENV"`,
 		OptType:        types.String,
 		CustomSetValue: SetConfigOptionStellarPrivateKey,
 		ConfigKey:      configKey,
-		Required:       true,
+		Required:       false,
 	}
 }
 
-func ChannelAccountEncryptionPassphraseOption(configKey *string) *config.ConfigOption {
+func DistributionAccountSignatureClientProviderOption(configKey *signing.SignatureClientType) *config.ConfigOption {
```

Comment:
Shall we standardize the config and function names?
```suggestion
func DistributionAccountSignatureProviderOption(configKey *signing.SignatureClientType) *config.ConfigOption {
```